src/App.js

```diff
@@ -1,7 +1,33 @@
+import React, { Component } from "react";
+import ProductDisplay from "./components/ProductDisplay";
+import Cart from "./components/Cart";
+import Checkout from "./components/Checkout";
+import formatPrice from "./helpers/formatPrice"
 import "./App.css";
 
-const App = () => {
-  return <h1>Hello, world!</h1>;
-};
+export default class App extends Component {
+  state = { cart: []}
 
-export default App;
+  addToCart = (product) => {
+    this.setState((prevState)=>({
+      cart: [...prevState.cart, product]
+    }))
+  }
+  render() {
+    const {cart} = this.state
+    const { products } = this.props;
+    return (
+      <div class="App">
+        <div className="leftSide">
+          <h1>My Garage Sale</h1>
+          <ProductDisplay products={products} addToCart={this.addToCart} formatPrice = {formatPrice}/>
```

Comment:
Convention says you should define `formatPrice` inside the class as a class method, then you could reference it as `this.formatPrice`. But the way you're doing it here, defining it outside the class, is great too.